sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/KeyVaultBackupAsyncClient.java

```diff
@@ -340,20 +340,20 @@ private static LongRunningOperationStatus toLongRunningOperationStatus(String op
 
         return clientImpl.fullRestoreOperationWithResponseAsync(vaultUrl, restoreOperationParameters,
             context.addData(AZ_TRACING_NAMESPACE_KEY, KEYVAULT_TRACING_NAMESPACE_VALUE))
-            .doOnRequest(ignored -> logger.info("Restoring from location - {}\\{}", blobStorageUrl, folderName))
-            .doOnSuccess(response -> logger.info("Restored from location - {}\\{}", blobStorageUrl, folderName))
+            .doOnRequest(ignored -> logger.info("Restoring from location - {}{}", containerUrl, folderName))
+            .doOnSuccess(response -> logger.info("Restored from location - {}{}", containerUrl, folderName))
             .doOnError(error ->
-                logger.warning("Failed to restore at location - {}\\{}", blobStorageUrl, folderName, error))
+                logger.warning("Failed to restore from location - {}{}", containerUrl, folderName, error))
```

Comment:
nit: simplify to below?
```suggestion
                logger.warning("Failed to restore from location - {}", backupFolderUrl, error))
```